ftplugin/php.vim

```diff
@@ -9,44 +9,36 @@ let g:phpactorInitialCwd = getcwd()
 let g:phpactorCompleteLabelTruncateLength=50
 let g:_phpactorCompletionMeta = {}
 
-if !exists('g:phpactorPhpBin')
-    ""
-    " Path to the PHP binary used by Phpactor
-    let g:phpactorPhpBin = 'php'
-endif
-
-if !exists('g:phpactorBranch')
-    ""
-    " The Phpactor branch to use when calling @command(PhpactorUpdate)
-    let g:phpactorBranch = 'master'
-endif
-
-if !exists('g:phpactorOmniAutoClassImport')
-    ""
-    " Automatically import classes when using VIM native omni-completion
-    let g:phpactorOmniAutoClassImport = v:true
-endif
-
-if !exists('g:phpactorCompletionIgnoreCase')
-    ""
-    " Ignore case when suggestion completion results
-    let g:phpactorCompletionIgnoreCase = 1
-endif
-
-if !exists('g:phpactorQuickfixStrategy')
-    ""
-    " Function to use when populating a list of code references. The default
-    " is to use the VIM quick-fix list.
-    let g:phpactorQuickfixStrategy = 'phpactor#quickfix#vim'
-endif
-
-if !exists('g:phpactorInputListStrategy')
-    ""
-    " Function to use when presenting a user with a choice of options. The default
-    " is to use the VIM inputlist.
-    let g:phpactorInputListStrategy = 'phpactor#input#list#inputlist'
-endif
-
+""
+" Path to the PHP binary used by Phpactor
+let g:phpactorPhpBin = get(g:, 'phpactorPhpBin', 'php')
+
+""
+" The Phpactor branch to use when calling @command(PhpactorUpdate)
+let g:phpactorBranch = get(g:, 'phpactorBranch', 'master')
+
+""
+" Automatically import classes when using VIM native omni-completion
+let g:phpactorOmniAutoClassImport = get(g:, 'phpactorOmniAutoClassImport', v:true)
+
+""
+" Ignore case when suggestion completion results
+let g:phpactorCompletionIgnoreCase = get(g:, 'phpactorCompletionIgnoreCase', 1)
+
+""
+" Function to use when populating a list of code references. The default
+" is to use the VIM quick-fix list.
+let g:phpactorQuickfixStrategy = get(g:, 'phpactorQuickfixStrategy', 'phpactor#quickfix#vim')
+
+""
+" Function to use when presenting a user with a choice of options. The default
+" is to use the VIM inputlist.
+let g:phpactorInputListStrategy = get(g:, 'phpactorInputListStrategy', 'phpactor#input#list#inputlist')
+
+""
+" When jump to the line of a file displayed in any existing window
+" reuse this window to avoid have more than one view of the same file.
+" The default is false.
```

Comment:
When jumping to a definition: if the file containing the definition is open in a window, jump to that window instead of switching buffers (?)

Reply:
I cannot judge what version is more proper. Feel free to choose the better in your opinion.

Reply:
yes, if what I said above is technically correct, then it could be better :+1: